RealmResultsController.xcodeproj/project.pbxproj

```diff
@@ -176,6 +204,8 @@
 		54D2990E1C523536710E0345 /* Frameworks */ = {
 			isa = PBXGroup;
 			children = (
+				83328AD01CD8C96400B70630 /* Realm.framework */,
+				83328AD11CD8C96400B70630 /* RealmSwift.framework */,
```

Comment:
Could you please put them in different folders? `iOS` and `tvOS` inside the `Frameworks` folder 🙏